src/app/tests/suites/certification/Test_TC_TSTAT_2_1.yaml

```diff
@@ -730,3 +730,97 @@ tests:
           value: 0
           constraints:
               type: enum8
+
```

Comment:
why are steps not matching exactly with the test plan. If you see above steps, Step 11 from test plan is missing and coded as step 12, since then all steps in the script are not aligned with the test plan.

Reply:
This was existing mis-numbering of the steps. I just added to it without realizing the numbers were off. I assumed the first 2 steps are commision to dut and something else. anyways fixed.